h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -840,7 +856,7 @@ public void init(boolean expensive) {
       if (bc.hasBounds() && _parms._early_stopping)
         warn("beta constraint and early_stopping", "if both are enabled may degrade model performance.");
       _state.setBC(bc);
-      if(hasOffsetCol() && _parms._intercept) { // fit intercept
+      if(hasOffsetCol() && _parms._intercept && !ordinal.equals(_parms._family)) { // fit intercept
```

Comment:
this shouldn't be needed because hasOffsetCol for ordinal should be false

Reply:
Michalk:  Right now the ordinal family does not preclude using the offset column.  I think you want me to throw an error if family == ordinal and hasOffsetCol() then.  Let me add that.